cfme/ansible_tower/explorer.py

```diff
@@ -121,16 +147,27 @@ class AnsibleTowerSystemsCollection(BaseCollection):
 
 
 @attr.s
-class AnsibleTowerJobTemplate(BaseEntity):
-    pass
+class AnsibleTowerJobTemplate(BaseEntity, Taggable):
+    name = attr.ib()
+    details_view = TowerExplorerJobTemplateDetailsView
 
 
 @attr.s
-class AnsibleTowerJobTemplatesCollection(BaseCollection):
+class AnsibleTowerJobTemplatesCollection(BaseCollection, TaggableCollection):
     ENTITY = AnsibleTowerJobTemplate
 
+    def all(self):
+        """Return entities for all items in Ansible Job templates collection"""
+        view = navigate_to(self, "All")
```

Comment:
Question: Is it possible to change this to an API call or DB query?

Reply:
@john-dupuy that's a good idea. I don't see the rest methods, unfortunately. Let me know, if I missed it.